docs/CHANGELOG.rst

```diff
@@ -24,6 +24,8 @@ Changed
 * Set core resource requirement in ``cuffnorm`` process to 1
 * Add sample descriptor to ``prepare_expression`` test function
 * Add access to sample field of data objects in processes
+* Add ``HISAT2`` and ``RSEM`` to ``resolwebio/rnaseq`` Docker image
```

Comment:
Reference to this image is now under added and changed section, all within a single release. I suggest you omit this, or modify the original change-log entry .